build/Targets/Imports.targets

```diff
@@ -245,6 +249,16 @@
     </SuggestedBindingRedirects>
   </ItemGroup>
 
+  <!-- 
+       Work around some UI delays in the new project system
+
+       https://github.com/dotnet/project-system/issues/2297
```

Comment:
I will test this out later today and if Roslyn performs fine will remove the item.